sources/packages/web/src/components/generic/DetailHeader.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <span v-for="(headerName, index) in Object.keys(headerMap)" :key="headerName">
+    <span class="label-bold-normal">{{ headerName }} : </span>
+    <span class="label-value-normal"> {{ headerMap[headerName] }} </span>
+    <!-- TODO: v-divider vertical must be used here once Vuetify 3 vertical divider is working.  -->
```

Comment:
Is this TODO still needed? The `v-divider` seems to be working from Vue 3 website.

Reply:
v-divider is working only as a horizontal divider. vertical divider not working(when vertical attribute is used)